build/cmake/CMakeLists.txt

```diff
@@ -24,14 +24,7 @@ string(REGEX REPLACE "^#define LZ4_VERSION_RELEASE +([0-9]+) +.*$" "\\1" LZ4_VER
 set(LZ4_VERSION_STRING "${LZ4_VERSION_MAJOR}.${LZ4_VERSION_MINOR}.${LZ4_VERSION_RELEASE}")
 mark_as_advanced(LZ4_VERSION_STRING LZ4_VERSION_MAJOR LZ4_VERSION_MINOR LZ4_VERSION_RELEASE)
 
-if("${CMAKE_VERSION}" VERSION_LESS "3.0")
-  project(LZ4 C)
-else()
-  cmake_policy (SET CMP0048 NEW)
```

Comment:
Question : is it now useless to specify this line ?
`cmake_policy (SET CMP0048 NEW)`

Reply:
Yeah, the policy is from 3.0 and with minimum required >= 3.0 it's automatically NEW